armstrong-react/source/components/form/inputs/tagInput.tsx

```diff
@@ -131,6 +132,13 @@ const TagInputRef: React.RefForwardingComponent<ITagInput, ITagInputProps> = (pr
     }
   }, [tags, filterSuggestions, notifySuggestionsChange])
 
+  const addSuggestion = React.useCallback(() => {
+    const suggestion = suggestions[suggestionIndex]
```

Comment:
Does suggestionIndex need to be a dep? :)

Reply:
yes!